test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/EventObjectTest.scala

```diff
@@ -0,0 +1,49 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2016, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib.util
+
+import org.junit.Test
+import org.junit.Assert._
+
+import java.util.EventObject
+
+class EventObjectTest {
+  @Test def getSource(): Unit = {
+    val src = new AnyRef
+    val e = new EventObject(src)
+    assertSame(src, e.getSource)
+  }
```

Comment:
As `source` is a protected `var` I would check that we can modify it from a class that extends from `EventObject`. Mainly to check that the JVM behaves the same way.